src/types/venues.ts

```diff
@@ -111,6 +111,7 @@ export interface Venue_v2
 
 export interface Venue_v2_Base {
```

Comment:
Yep, `WithSlug` could probably be useful in many places, but then, complicating types for a single field... 🤔

Reply:
It's 50/50. It might come in useful but in the end this field should be a part of the venue and it should be required everywhere the type is being used.

Reply:
It is also a world that has a slug, hence the idea of `WithSlug`. We should keep an eye of possible third concept also acquiring this wormy thing in the future.